index.html

```diff
@@ -0,0 +1,44 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
```

Comment:
It is better to add a description on each page's contents for the sake of clarity! For example, <!-- Header Starts Here --> Comments speak loudly and make easier and clean your code for other clients to use.